account_move_name_sequence/readme/CONTRIBUTORS.md

```diff
@@ -5,3 +5,5 @@
   - Francisco Luna \<<fluna@vauxoo.com>\>
 - [Factor Libre](https://www.factorlibre.com):
   - Rodrigo Bonilla Martinez \<<rodrigo.bonilla@factorlibre.com>\>
+- [XCG Consulting](https://xcg-consulting.fr):
+  - Houzéfa Abbasbhay
```

Comment:
I remember a discussion regarding when contributors should be added. If I remember correctly, main contributors are considered as such when they perform big changes or structural ones, or they are responsible for more than the 5% of the code. If that's the case, this contribution wouldn't be candidate for that.

Reply:
I have always done this in my PRs, small or large ones, eg in https://github.com/OCA/project/pull/1215

Of course I don't mind removing it if it creates confusion

Reply:
Removed